### DISCUSSION TITLE
Add ledger wallet to a none ledger profile

### DISCUSSION THREAD

**Discussion by zarte13** [iotaledger/firefly#3002] [Features / Enhancements]
Adding ledger wallet(s) to a none ledger profile would be very useful
![image](https://user-images.githubusercontent.com/34402486/164467413-a62768b5-248d-4a14-9451-aeb6b402592e.png)

**Comment by MiKeF05** [comment]
This is not possible because the ledger has a different mnemonic. All wallets inside a profile are derived from one mnemonic.
